File-Formats/PDF/Convert-HTML-To-PDF/mac.md

```diff
@@ -9,75 +9,73 @@ keywords: Assemblies
+6. A default action method named Index will be present in HomeController.cs. Right click on Index method and select Go To View where you will be directed to its associated view page Index.cshtml.
 
-Add a new button in the Index.cshtml as shown below.
+   Add a new button in the Index.cshtml as shown below.
 
-{% highlight c# tabtitle="C#" %}
+   {% highlight c# tabtitle="C#" %}
 
-@{Html.BeginForm("ExportToPDF", "Home", FormMethod.Post);
-{
-<div>
-    <input type="submit" value="Convert PDF" style="width:150px;height:27px" />
-</div>
-}
-Html.EndForm();
-}
+   @{Html.BeginForm("ExportToPDF", "Home", FormMethod.Post);
+   {
+     <div>
+      <input type="submit" value="Convert PDF" style="width:150px;height:27px" />
+     </div>
+   }
+   Html.EndForm();
+   }
 
-{% endhighlight %}
+   {% endhighlight %}
 
-Add a new action method ExportToPDF in HomeController.cs and include the below code snippet to convert HTML to PDF file and download it.
+7. Add a new action method ExportToPDF in HomeController.cs and include the below code snippet to convert HTML to PDF file and download it.
 
-{% highlight c# tabtitle="C#" %}
+   {% highlight c# tabtitle="C#" %}
 
-public IActionResult ExportToPDF()
-{
-        //Initialize HTML to PDF converter.
-        HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
+        public IActionResult ExportToPDF()
+        {
+            //Initialize HTML to PDF converter.
+            HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
+            //Convert URL to PDF.
+            PdfDocument document = htmlConverter.Convert("https://www.syncfusion.com");
+            MemoryStream stream = new MemoryStream();
+            document.Save(stream);
+            //Download the PDF document in the browser
+            return File(stream.ToArray(), System.Net.Mime.MediaTypeNames.Application.Pdf, "HTML-to-PDF.pdf");
+        }
 
-        //Convert URL to PDF.
-        PdfDocument document = htmlConverter.Convert("https://www.syncfusion.com");
-        MemoryStream stream = new MemoryStream();
-        document.Save(stream);
-       //Download the PDF document in the browser
-        return File(stream.ToArray(), System.Net.Mime.MediaTypeNames.Application.Pdf, "HTML-to-PDF.pdf");
+   {% endhighlight %}
 
-}
-
-{% endhighlight %}
-
-Right click the project and select Build.
+8. Right click the project and select Build.
 ![Mac_step5](htmlconversion_images/mac_step5.png)
 
-After Build succeeded. Run the application.
+9. After Build succeeded. Run the application.
 ![Mac_step6](htmlconversion_images/mac_step6.png)
 
-A complete working sample can be downloaded from [Mac-HTML-to-PDF-Demo.zip] (https://www.syncfusion.com/downloads/support/directtrac/general/ze/Mac-HTML-to-PDF-Demo514297355)
+10. A complete working sample can be downloaded from [Mac-HTML-to-PDF-Demo.zip](https://www.syncfusion.com/downloads/support/directtrac/general/ze/Mac-HTML-to-PDF-Demo514297355).
```

Comment:
Remove the numbering and add this line in last (after output document screenshot)

Reply:
Changed the content